players.md

```diff
@@ -0,0 +1,85 @@
+| **Vorbis**              | ✅                                                                                                             | ❌                                                                           |
+| **WavPack**             | ✅                                                                                                             | ❌                                                                           |
+| **WMA (Lossless, Pro)** | ✅                                                                                                             | ❌                                                                           |
+
+---
+
+## Video Support
+
+| Video Codec             | Swiftfin (VLCKit)                                                                                               | Native (AVKit)                                                               |
+|-------------------------|-----------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------|
+| **AV1**                 | ✅                                                                                                             | ❌                                                                           |
+| **H.263**               | ✅                                                                                                             | ❌                                                                           |
+| **H.264**               | ✅                                                                                                             | ✅                                                                           |
+| **H.265 (HEVC)**        | ✅                                                                                                             | ✅                                                                           |
+| **MJPEG**               | ✅                                                                                                             | ❌                                                                           |
+| **MPEG1/2/4**           | ✅                                                                                                             | ✅ (MPEG4 only)                                                              |
+| **VC1**                 | ✅                                                                                                             | ❌                                                                           |
+| **VP9**                 | ✅                                                                                                             | ❌                                                                           |
+
+---
+
+## Subtitle Support
```

Comment:
Since subtitle selection isn't working on Native (AFAIK?), it might be good to call that out specifically in the subtitle support section.